Sources/SwiftDocC/Model/TaskGroup.swift

```diff
@@ -93,6 +93,13 @@ struct ExtractLinks: MarkupRewriter {
                 guard let paragraph = item.child(at: 0) as? Paragraph,
                     paragraph.childCount >= 1 else { return true }
                 
+                // Check for trailing invalid content.
+                let containsInvalidContent = paragraph.children.dropFirst().contains(where: { child in
```

Comment:
Nit: Just FYI, Swift has a more recent syntax for trailing closures like this that reads a little nicer.

```suggestion
                let containsInvalidContent = paragraph.children.dropFirst().contains { child in
```

(and then delete the trailing parentheses on line 101)

Reply:
FWIW `contains(where:)` is one of the cases where I find that the argument labels helps make the closure read better